### PR TITLE
doc(functions): Update README.md

### PR DESCRIPTION
**Pull request by fivetran-abhijeet**
Updated the ReadME file of our Cloud Functions GitHub repo.

Closes https://fivetran.atlassian.net/browse/RD-77235

### CODE REVIEW

**Review by fivetran-meera — APPROVED** [review]

File: README.md
Comment (AK-Fivetran) [inline README.md:5]:
```suggestion
## Fivetran Cloud Function Connectors
```

File: README.md
Comment (AK-Fivetran) [inline README.md:7]:
```suggestion
[Function connectors](https://fivetran.com/docs/functions) allow you to code a custom data connector as an extension of Fivetran. If you have a custom data source or a private API, use our Function connectors to build robust serverless ELT data pipelines. You only have to write the cloud function to extract the data from your source. Fivetran will load and transform the data in your destination. 
```

**Review by AK-Fivetran — COMMENTED** [review]
The language looks good overall. I've suggested a couple of edits. I see you're still waiting for some SME approvals. I can't tell which ones are optional and which ones are required, so I will wait to give final approval until they all approve or you let me know which ones are optional.

File: README.md
Comment (fivetran-shashanktripathi) [inline README.md:7]: >Fivetran will load and transform the data in your destination

I think we should just say "Fivetran will load the data in your destination"

>If you have a custom data source or a private API

There can be more use-cases. maybe we can say "For example, If you have a custom data source or a private API"

**Review by fivetran-shashanktripathi — APPROVED** [review]
Looks good. 2 suggestions

File: README.md
Comment (fivetran-abhijeet) [reply README.md:7]: @fivetran-shashanktripathi I re-used our already approved content from the Functions page. Here is a screenshot for your reference. 
image

Comment (AK-Fivetran) [reply README.md:7]: @fivetran-abhijeet I think as a follow-up, we should apply the suggestion "For example, If you have a custom data source or a private API" both here and in the Functions overview doc to make it clear there could be other use cases. Underlining the open-ended nature of function connectors seems important.

But, this can be a tiny followup rather than holding us up here.

**Review by AK-Fivetran — APPROVED** [review]
Approved. Please note my request for a small followup later.

File: README.md
Comment (fivetran-abhijeet) [reply README.md:7]: sure @AK-Fivetran. I will create a follow-up.

**Pull request merged**
